### ISSUE TITLE
Errors when starting computer

### ISSUE THREAD

**Issue by ILoveSkilledCoders** [Open-Shell/Open-Shell-Menu#286]
Hi there,

So I've been using this for years, & I hate Win 10, but the Asus laptop I bought sevaral years ago came with Win 8 & of course upgraded to Win 10.

I keep getting these errors re: Classic Shell even after my website coder helped me configure it.

Can you tell me how to get rid of them & make sure that CS starts when I restart?

Thanks

First image, I press Ok & then 
![ClassicShellError](https://user-images.githubusercontent.com/41310815/72489145-d9f1db00-381b-11ea-862c-451f32a2354c.png) 

2nd image

![ClassicShellError2](https://user-images.githubusercontent.com/41310815/72489167-eaa25100-381b-11ea-9196-209b26621212.png)

**Comment by blackcrack** [comment]
1. witch Version of Win10,
2. which number of the Workstation in your office
3. which version of the Classic shell who you have installed ?

best regards
Blacky

**Comment by RegisStGelais** [comment]
You  probably still use Classic Shell. It is a known issue with that version that is no longer maintained. Download an install Open Shell. It is the rebooted open source version.
You can download the current version of Open Shell from https://github.com/Open-Shell/Open-Shell-Menu

**Comment by ILoveSkilledCoders** [comment]
[ClassicShellError-SquirrelSetup.log](https://github.com/Open-Shell/Open-Shell-Menu/files/4074273/ClassicShellError-SquirrelSetup.log)

@RegisStGelais Oh great, thanks for that.

Unfortunately it didn't work when I attempted to run it. 

I can't stand Win 10. Even double clicking for me doesn't work, but I did tell it to open by right clicking.

Here's the log file.

Thanks

**Comment by ge0rdi** [comment]
@ILoveSkilledCoders 
What exactly are you trying to install?
Provided log doesn't seem to be related to `Open-Shell` in any way.

**Comment by ILoveSkilledCoders** [comment]
LOL, so my website coder helped me & you never gave me the direct link, so it turns out I was DLing Github.

Anyway, after DLing the RIGHT software, I had problems.

First, there was an error, but the exe continued to install with the error message in the background. 

I closed both windows & tried again. Then it hung my computer & disappeared. I'm using remote view since there's no desk where my laptop is & I can't bend over, so I had to go to the laptop & check again.

First error - https://mega.nz/#!l0lTGS5R!I7pj3S4eyoS-cQkAT6ncAkmC1Pps6lyFUAa2XojAzq4
The 2nd time I got this image (see pic)
![OpenShellError](https://user-images.githubusercontent.com/41310815/72652163-4987d780-398e-11ea-8bfe-13495045e9ca.png)

And I tried several times, I even DLed it again & got the same error.

Thanks
